.github/workflows/staging-push.yml

```diff
@@ -52,6 +52,8 @@ jobs:
 
       - name: Jira query
         run: python ./__main__.py --report-type jira-qa-requests
+      - name: Jira query
```

Comment:
I am changing this to Jira qa-needed query, similar for L53 to qa-requests so that when looking at the github action steps is clearer which one it is